msvc/libmonoutils.vcxproj.filters

```diff
@@ -426,9 +417,9 @@
     <ClInclude Include="..\mono\utils\os-event.h">
       <Filter>Header Files</Filter>
     </ClInclude>

```

Comment:
Unrelated changes.

Reply:
Also done by VS environment when project file was changed, I could manually revert that change outside the editor as well.